paypal/paypal/helper.php

```diff
@@ -86,7 +86,9 @@ public function Storelog($name, $logdata)
 	 */
 	public function validateIPN($data, $componentName)
 	{
-		$url              = self::buildPaypalUrl();
```

Comment:
so we not using `buildPaypalUrl` anymore?
can we use it `buildPaypalUrl` and modify it as needed? @ankush-maherwal

Reply:
We are using function 'buildPaypalUrl', this function creates URL for posting data to PayPal for payment.

As PayPal has provided a separate server for IPNs we are not using function 'buildPaypalUrl' to create IPN URL.

Also, we are using IPN URL only in one place hence not created a separate function for this.